packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -164,13 +164,13 @@ const Button = ({
 
   return (
     <>
-      {tooltipText && (
+      {tooltipText && !isDisabled && (
         <Tooltip ariaType="description" placement="top" text={tooltipText}>
           <MuiPropsContext.Consumer>{renderButton}</MuiPropsContext.Consumer>
         </Tooltip>
       )}
 
-      {!tooltipText && renderButton(muiProps)}
+      {!(tooltipText && !isDisabled) && renderButton(muiProps)}
```

Comment:
I'm not a regular contributor here, but I'd recommend this for readability (rather than having to mentally do the double-negative):

`{ (!tooltipText || isDisabled) && renderButton(muiProps) }`

Reply:
That's a big improvement, I'll make that change! 🙌

Reply:
And we welcome all suggestions, no matter how regular or irregular a contributor you are. The more, the better! 🚀

Reply:
I would even swap it more:
```ts
{ (isDisabled || !tooltipText) && renderButton(muiProps) }
```

Put the positive check first.